testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketConnectionAttemptTest.java

```diff
@@ -122,7 +113,8 @@ public void testConnectCancellation() throws Throwable {
             }
         }
 
-        assumeThat("The connection attempt to " + BAD_HOST + " does not time out.", badHostTimedOut, is(true));
+        assumeThat("The connection attempt to " + SocketTestPermutation.BAD_HOST + " does not time out.",
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'SocketTestPermutation.BAD_HOST' due to existing static import 'io.netty.testsuite.transport.socket.SocketTestPermutation.BAD_HOST' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)